### PR TITLE
Seer performance badges for attribute updates and render

### PR DESCRIPTION
**Pull request by ibgreen**
screen shot 2017-06-19 at 8 56 40 am

### CODE REVIEW

File: src/lib/stats.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    formatted = `${ms.toFixed(1)}ms`;
+  } else if (ms < 1000) {
+    formatted = `${ms.toFixed(0)}ms`;
+  } else {
+    formatted = `${(ms / 1000).toFixed(2)}s`;
+  }
+  return formatted;
+}
+
+export function leftPad(string, length = 8) {
```
Comment (Apercu) [inline src/lib/stats.js]: haha, better be safe ;)

File: test/index.js [hunk 2]
```diff
@@ -24,4 +24,4 @@ import './lib';
 import './utils';
 import './experimental';
 import './deprecated';
-import './debug/seer-integration';
+import './debug/seer-integration.spec';
```
Comment (Apercu) [inline test/index.js]: why changing this? we are already in the tests directory

File: src/lib/layer-manager.js [hunk 3]
```diff
@@ -70,10 +71,11 @@ export default class LayerManager {
     Object.seal(this);
 
     /**
-     * Set an override on the specified property and update the layers
-     */
+     import {setPropOverrides, layerEditListener, initLayerInSeer, updateLayerInSeer}
```
Comment (Apercu) [inline src/lib/layer-manager.js]: what's this comment?

File: test/index.js (hunk 2 above)
Comment (ibgreen) [inline test/index.js]: > why changing this? we are already in the tests directory

Every other file in the test directory follows this `.spec` convention.

This convention is pretty common in the JS community. FWIW, it does have the advantage of easily distinguishing the two files e.g. in sublime tabs or when do command-open in the Chrome debugger.

File: src/lib/layer-manager.js (hunk 3 above)
Comment (ibgreen) [inline src/lib/layer-manager.js]: > what's this comment?

Yes looks strange, will remove.

**Comment by ibgreen** [comment]
> Very nice, but isn't the formatting of the ms badge strange? shouldn't we only see the number?

Yes, we can play around with it, but I prefer to leave as-is for now.

I want to see the count of how many updates have happened (often more important than the number). Also wanted the user to have a chance of understanding what it means. If we just have two numbers, it will also be confusing to the user. 

If I could provide a tooltip for these badges, with additional data might be the best option...

File: test/index.js (hunk 2 above)
Comment (Apercu) [inline test/index.js]: sure

**Pull request merged**